orga/translations/en.json

```diff
@@ -670,18 +670,24 @@
     },
     "students-sup-import": {
       "title": "Import students",
+      "description": "What kind of import do you want to make?",
```

Comment:
traduction officielle : "What type of import would you like to use?"

Reply:
done